src/components/CreateIconButton/CreateIconButton.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  fade,
+  IconButton,
+  IconButtonProps,
+  makeStyles,
+} from '@material-ui/core';
+import { Add } from '@material-ui/icons';
+import clsx from 'clsx';
+import { FC } from 'react';
+import * as React from 'react';
+
+const useStyles = makeStyles(({ palette }) => ({
+  createButton: {
+    backgroundColor: palette.error.main,
+    color: palette.common.white,
+    '&:hover': {
+      backgroundColor: fade(palette.error.main, 0.5),
+    },
+  },
+}));
+
+export const CreateIconButton: FC<IconButtonProps> = ({
```

Comment:
MUI actually has a `Fab` button that is exactly this. I've also wrapped it to provide loading state and error color. Sorry for the bait and switch 😞  I didn't remember it when we were talking about this last.

Reply:
This is switched out to the Fab component